test/WebJobs.Script.Tests/app.config

```diff
@@ -1,39 +1,53 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
+  <connectionStrings>

```

Comment:
Recommend removing this, and instead use env vars for all of this. Otherwise, you'll accidentally check in your connection strings